chart/otomi/Chart.yaml

```diff
@@ -4,7 +4,7 @@ description: A Helm chart for installing otomi in Kubernetes
 home: https://otomi.io/
 icon: https://otomi.io/img/otomi-logo.svg
 type: application
-version: 0.3.CHART_PATCH_PLACEHOLDER
+version: 0.3.0 #CHART_PATCH_PLACEHOLDER
```

Comment:
CHART_PATCH_PLACEHOLDER is not needed anymore?

Reply:
oops...hate that mechanism...have to find a workaround